### PR TITLE
Fix for upcoming change to Discord voice nodes

### PR DESCRIPTION
**Pull request by HcgRandon**
Discord is going to be deploying an update to their voice nodes. This change means that the endpoint and the IP found in Voice OPCode 2 are no longer the same.

This change now correctly uses the IP from OPCode 2. Instead of DNS resolving the IP of the endpoint.

### CODE REVIEW

**Comment by HcgRandon** [comment]
~~Changing the Voice OPCode Constants Might of broke stuff I forgot to check. Abal would know off hand. I need to check~~ Looks like only place that uses VoiceOPCodes is VoiceConnection

**Comment by abalabahaha** [comment]
How's this doing on Ayana prod? ~~I can't remember why I was caching the DNS query, but it should have been non-trivial.~~ I lied, no need for DNS.

Also, what's with the unused Voice3 constants?

**Comment by HcgRandon** [comment]
This is only tested on my dev bot ATM. I'm going to push it to Ayana PRO shortly.

As for DNS it is no longer needed here. Because it will always be an IP per Discord Dev
image

**Comment by HcgRandon** [comment]
If you wish to test this change, Simply use Japan voice servers. Discord has deployed this change there. Prior to this fix Eris will fail to connect

File: lib/Constants.js [hunk 1]
```diff
@@ -79,10 +79,10 @@ module.exports.Permissions = {
 module.exports.VoiceOPCodes = {
     IDENTIFY:            0,
     SELECT_PROTOCOL:     1,
-    HELLO:               2,
+    READY:               2,
     HEARTBEAT:           3,
     SESSION_DESCRIPTION: 4,
-    SPEAKING:            5
+    SPEAKING:            5,
```
Comment (abalabahaha) [inline lib/Constants.js]: Trailing comma?

**Pull request merged**